PlayerControls/sources/Command.swift

```diff
@@ -0,0 +1,32 @@
+//  Copyright © 2017 One by AOL : Publishers. All rights reserved.
+
+public typealias Command = CommandWith<Void>
+
+public struct CommandWith<T> {
+    private var action: (T) -> Void
```

Comment:
If use `perform` instead `action` struct will become more elegant

Reply:
Command with action -> Command with perform
Don't you think that with action it looks much more readable?

Reply:
If `action` will be renamed with `perform` and not `private`, then `initi` and `perform` methods can be removed too

Reply:
Well, it wouldn't change anything. Are there any other suggestions?